pkg/testgridanalysis/testidentification/ocp_variants.go

```diff
@@ -96,31 +94,13 @@ var (
 		// https://bugzilla.redhat.com/show_bug.cgi?id=2003646
 		"periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-workers-rhel8",
 
-		// https://bugzilla.redhat.com/show_bug.cgi?id=20000754
-		"periodic-ci-openshift-release-master-ci-4.10-e2e-openstack-parallel",
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-openstack-proxy",
-		"release-openshift-ocp-installer-e2e-aws-upi-4.10",
-		"release-openshift-ocp-installer-e2e-aws-upi-4.9",
-
 		// https://bugzilla.redhat.com/show_bug.cgi?id=2008201
 		"periodic-ci-openshift-release-master-nightly-4.10-e2e-openstack-az",
 
-		// https://bugzilla.redhat.com/show_bug.cgi?id=2007850
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-fips-serial",
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-azure-fips-serial",
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-gcp-fips-serial",
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-openstack-fips",
-
 		// https://bugzilla.redhat.com/show_bug.cgi?id=2007692
 		"release-openshift-origin-installer-old-rhcos-e2e-aws-4.9",
 		"release-openshift-origin-installer-old-rhcos-e2e-aws-4.10",
 
-		// https://bugzilla.redhat.com/show_bug.cgi?id=2008128
-		"release-openshift-origin-installer-e2e-azure-shared-vpc-4.9",
-		"release-openshift-origin-installer-e2e-azure-shared-vpc-4.10",
-		"release-openshift-origin-installer-e2e-aws-shared-vpc-4.9",
-		"release-openshift-origin-installer-e2e-aws-shared-vpc-4.10",
-
```

Comment:
More going out than going in, awesome.

Would be cool if someday sippy could inform us when never-stable jobs with an associated bug has gone verified.